chunky/src/java/se/llbit/chunky/ui/controller/ChunkyFxController.java

```diff
@@ -734,6 +733,24 @@ public File getSceneFile(String fileName) {
         PersistentSettings.setSppTargetDefault(scene.getTargetSpp()));
   }
 
+  public void confirmAndClose(Event event) {
+    Alert confirmQuit = Dialogs.createAlert(Alert.AlertType.CONFIRMATION);
+    confirmQuit.setTitle("Quit Chunky");
+    confirmQuit.setHeaderText("You have unsaved changes in your scene.");
```

Comment:
Until we have #1709, I'd suggest reworing this so that users don't think this is a bug:

```suggestion
    confirmQuit.setHeaderText("You may have unsaved changes in your scene.");
```

Reply:
sounds good